pymtl/tools/translation/verilog_behavioral_test.py

```diff
@@ -20,6 +20,7 @@
 from ..simulation.SimulationTool_comb_test   import *
 from ..simulation.SimulationTool_mix_test    import *
 from ..simulation.SimulationTool_transl_test import *
+from ..simulation.dynamic_ast_test           import *
```

Comment:
So this import from simulation tool does work? I mean did you check if the test case is actually tested?

Reply:
Yes -- I specified the name of the test with -k and pytest can successfully discover and run the simulation and translation tests.